components/x-teaser-timeline/src/lib/transform.js

```diff
@@ -19,7 +19,11 @@ const groupItemsByLocalisedDate = (
 
 	items.forEach((item, index) => {
 		const localDateTime = getLocalisedISODate(item.publishedDate, timezoneOffset)
-		const localDate = getDateOnly(localDateTime)
+		// N.B. This is to keep a previous hack that existed. Changing this may break existing dates.
```

Comment:
sorry Ben, we are not familiar with this codebase. To which hack are you referring?
the commit linked is just a refactor to be able to reuse `getDateOnly()`